tox.ini

```
@@ -2,16 +2,17 @@
 min_version = 4.11
 
 env_list =
-    python{39,310,311}-django4.2-wagtail{5.2,6.1,6.2}
-    python{310,311,312}-django5.0-wagtail{5.2,6.1,6.2}
-    python312-django5.1-wagtail{6.2,main}
+    python{39,310,311}-django4.2-wagtail{5.2,6.2,6.3,6.4}
+    python{310,311,312}-django5.0-wagtail{5.2,6.2,6.3,6.4}
+    python{312,313}-django5.1-wagtail{6.3,6.4,main}
```

```suggestion
    python{39,310,311}-django4.2-wagtail{5.2,6.3,6.4}
    python{312,313}-django5.1-wagtail{6.3,6.4,main}
```

with Django 5.0 out of support in about a month, may as well drop it from the matrix